### DISCUSSION TITLE
My font is comic sans when its supposed to be Inter

### DISCUSSION THREAD

**Discussion by elizzakai** [community/community#36828] [Other Feature Feedback, Questions, & Ideas] [answered]
Ok, so I keep seeing a 404 error on the Inter file when I inspect my GitHub pages portfolio site. Thus, instead of the font showing as Inter it shows as comic sans instead for some reason. Though, on local it shows up fine, no errors in the console. 

Maybe you guys could help me with this issue ? I think everything else works thus far.

**Link to repo: https://github.com/elizzakai/my_Portfolio
Github Pages site: https://elizzakai.github.io/my_Portfolio/**

**Comment by DenverCoder1** [comment, marked as answer]
When a font is not loaded, it will fall back to the next font in the font stack. In your case, that is "cursive" which is your default cursive font for your system (Comic Sans).

There are a few ways to load fonts.

For Google fonts, one way is using the following:

```html
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
```

Another way is with just CSS `@import`, for example:

```html
<style>
@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
</style>
```

You can find this by adding the fonts on fonts.google.com and opening the side panel.

Also note, since Inter appears to be a sans-serif font, you may want to create a fallback of "sans-serif" rather than "cursive" to avoid making it go to another style such as Comic Sans. (i.e. `font-family: 'Inter', sans-serif;`)

**Reply by elizzakai** [reply]
Ah okay! Thank you so much!!
